packages/shared-ux/page/analytics_no_data/impl/src/services.tsx

```diff
@@ -45,6 +45,7 @@ export const AnalyticsNoDataPageKibanaProvider: FC<AnalyticsNoDataPageKibanaDepe
 }) => {
   const value: Services = {
     kibanaGuideDocLink: dependencies.coreStart.docLinks.links.kibana.guide,
+    kibanaHasCustomBranding: dependencies.coreStart.customBranding.hasCustomBranding,
```

Comment:
This is not how the service definition should look like. We need to take the observable from `customBranding` service, add subscriber to it in `analytics_no_data_page` and then propagate the updated value further.